public/docs/ts/latest/guide/index.jade

```diff
@@ -1,10 +1,501 @@
+  1. [Metadata](#metadata)
+  1. [Data Binding](#data-binding)
+  1. [Directive](#directive)
+  1. [Service](#service)
+  1. [Dependency Injection](#dependency-injection)
+ 
+figure
+  img(src="/resources/images/devguide/intro/overview.png" alt="overview" style="margin-left:-40px;" )
+:marked
+  Learn these seven and we're on our way.
+
+.l-main-section
+<a id="component"></a>
+:marked
+  ## The Component
+figure
+  img(src="/resources/images/devguide/intro/hero-component.png" alt="Component" align="left" style="width:200px; margin-left:-40px;margin-right:10px" )
+:marked
+  A **Component** controls a patch of screen real estate that we could call a "view". 
+  The shell at the application root with navigation links, that list of heroes, the hero editor ... 
+  they're all views controlled by Components.
+  
+  We define a Component's "application logic" - what it does to support the view - inside a class.
+  The class interacts with the view through an API of properties and methods. 
+  
+  <a id="component-code"></a>
+  A `HeroListComponent`, for example, might have a `heroes` property that returns an array of heroes
+  that it acquired from a service.
+  It might have a `selectHero()` method that sets a `selectedHero` property when the user click on a hero from that list.
+  It might be a class like this:
+
++makeExample('intro/ts/app/hero-list.component.ts', 'class')
+:marked
+  Angular creates, updates, and destroys components as the user moves through the application.
+  The developer can take action at each moment in this lifecycle through optional [Lifecycle Hooks](#).
+.l-sub-section
+  :marked
+    Our example shows the service arriving via the class constructor.
+    Who calls that constructor? Who provides the service?
+    For the moment, have faith that Angular is going to deliver an
+    appropriate `HeroService` when we need it.
+
+.l-main-section
+<a id="template"></a>
+:marked
+  ## The Template
+figure
+  img(src="/resources/images/devguide/intro/template.png" alt="Template" align="left" style="width:200px; margin-left:-40px;margin-right:10px" )
+:marked
+  We define a Component's view with its companion **Template**. A template is a form of HTML
+  that tells Angular how to render the Component.
+
+  A template looks like regular HTML much of the time ... and then it gets a bit strange. Here is a
+  template for our `HeroList` Component
++makeExample('intro/ts/app/hero-list.component.html')
+:marked
+  We recognize `<h2>` and  `<div>`. 
+  But there's other markup that no one told us about in school.
+  What is`*ng-for`, <code>{&zwnj;{hero.name}}</code>, `(click)`, `[hero]`, and `<hero-detail>`?
+  These are examples of Angular's [template syntax](template-syntax.html). 
+  We will grow accustomed to that syntax and may even learn to love it.
+  We'll begin to explain it in a moment.
+  
+  Before we do, focus attention on the last line. 
+  The `<hero-detail>` tag is a custom element representing the `HeroDetailComponent`.
```

Comment:
I struggled with this.  Of course it isn't an element at all. But I'm targeting the novice and I felt that imprecision was acceptable in the service of learning.

It will be interesting to see what others say.